hnn_core/batch_simulate.py

```diff
@@ -16,15 +16,14 @@
 
 
 class BatchSimulate(object):
-    def __init__(self, set_params, net=jones_2009_model(), tstop=170,
-                 dt=0.025, n_trials=1, record_vsec=False,
-                 record_isec=False, postproc=False, save_outputs=False,
+    def __init__(self, set_params, net=None,
```

Comment:
For the default `net` object here, is there a specific reason why you changed Lines 19, 109, and 132 (corresponding to the later version of `batch_simulate.py`)? As far as I tested, both versions work the same: they use `jones_2009_model()` if nothing is passed, but they successfully use a passed `Network` if the user passes a different one. If there is no use-case where the changes to `net` in Lines 19, 109, and 132  are required, then would you please revert them to the original? The original is cleaner.